src/EFCore.PG/Extensions/NpgsqlUnaccentDbFunctionsExtensions.cs

```diff
@@ -0,0 +1,41 @@
+using System;
+using Microsoft.EntityFrameworkCore.Diagnostics;
+
+// ReSharper disable once CheckNamespace
+namespace Microsoft.EntityFrameworkCore
+{
+    /// <summary>
+    /// Provides Npgsql unaccent extension methods on <see cref="DbFunctions"/>.
+    /// </summary>
+    public static class NpgsqlUnaccentDbFunctionsExtensions
```

Comment:
The methods here can move into NpgsqlFullTextSearchDbFunctionsExtensions. We have dedicated files for cases where a lot of extensions are defined, but that isn't the case here.

Reply:
ok I moved them to the NpgsqlFullTextSearchDbFunctionsExtensions class